app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="log-in-section">
+  <h2 class="sign-in-h2">Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: {class: "log-in-form-container"}) do |f| %>
+    <div class="field">
+      <%= f.label :email %>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
```

Comment:
- [ ] Nice job making the signup/login page work 👌 However, this activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your login view:**        |  **Wireframe login view:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/batoorsohail/blog-app/assets/12745474/65b3440f-2ce0-4b58-b103-9e24e6847639"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">